src/components/tx/CreateTx/createTxForm.js

```diff
@@ -170,11 +168,11 @@ export const CreateTxForm = (props) => {
               />
             )}
             {/*
-                     <Divider />
-                     {addressBook.map((account) =>
-                     <MenuItem key={account.get('id')} value={account.get('id')} primaryText={account.get('id')} />
-                     )}
-                     */}
+                <Divider />
```

Comment:
eslint doenst touch code in comments, so this remains unchanged.

should probably delete it, since you know... version control =P